### PR TITLE
chore: add regex and log author data

### PR DESCRIPTION
**Pull request by cbernat**
- Add log for author in console (temporal for checking)
- Set regexp for author name in change request

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+        stage('Identification') {
+            steps {
+                sh 'echo Change request author: ${CHANGE_AUTHOR} '
+            }
+        }
         stage('Confirmation') {
             when {
                 beforeInput true
-                changeRequest author: 'dependabot-preview'
+                changeRequest author: '[\\w]*dependabot[\\w]*', comparator: "REGEXP"
```
Comment (andresmoschini) [inline Jenkinsfile]: Did you verify that this is not enough?

```suggestion
                changeRequest author: 'dependabot', comparator: "REGEXP"
```

Comment (andresmoschini) [reply Jenkinsfile]: BTW, `-` is a word character (`\w`)

Comment (cbernat) [reply Jenkinsfile]: Perhaps this suffice I'm not so sure. I guess I can test this approach with my user to be sure.

**Comment by cbernat** [comment]
test this please.

**Pull request merged**